lib/TikZImage.pm

```diff
@@ -219,4 +248,42 @@ sub draw {
 	return $data;
```

Comment:
There is a minor thing here also.  This is my mistake that I am going to ask you to fix.  This should probably use the external rm program (also defined in site.conf).

Reply:
I think all of these external programs (mv, cp, rm, mkdir, tar, and gzip) are here for non-linux operating system usage.  All of these are always available on linux and always in the path.  So there is no need to use the full path.